bedrock/mozorg/urls.py

```diff
@@ -89,7 +89,6 @@
     page('contact/spaces/portland', 'mozorg/contact/spaces/portland.html'),
     page('contact/spaces/san-francisco', 'mozorg/contact/spaces/san-francisco.html'),
     page('contact/spaces/taipei', 'mozorg/contact/spaces/taipei.html'),
-    page('contact/spaces/tokyo', 'mozorg/contact/spaces/tokyo.html'),
```

Comment:
Is it worth adding a redirect to the `/contact/spaces/` landing page, so this doesn't 404?

Also, should this definitely be removed from the spaces list? Has someone specifically asked for this to happen? It's not quite clear from Mitchell's email what the implications are for the Mozilla Japan space.